compiler/package.json

```diff
@@ -23,6 +23,10 @@
     "test": "esy test",
     "format": "esy format",
     "check-format": "esy check-format",
+    "parser:interpret": "esy b menhir src/parsing/parser.mly --unused-tokens --interpret",
+    "parser:interpret-error": "esy b menhir src/parsing/parser.mly --unused-tokens --interpret-error",
+    "parser:list-errors": "esy b menhir src/parsing/parser.mly --unused-tokens --list-errors > src/parsing/parser.messages.generated",
+    "parser:check-errors": "yarn parser:list-errors && esy b menhir src/parsing/parser.mly --unused-tokens --compare-errors src/parsing/parser.messages.generated --compare-errors src/parsing/parser.messages",
```

Comment:
Question about this. Should we move these commands into the esy.json file or keep them here?

Reply:
I would say you know best! If you think we should move it then I will.

Reply:
I don't have a strong feeling about it. I think all of the other ones just proxy to the esy.json scripts